include/s3select.h

```diff
@@ -15,6 +15,7 @@
 #include <boost/function.hpp>
 #include <boost/bind.hpp>
 #include <functional>
+#include <string_view>
```

Comment:
is it a must? 
s3select already using `string_view`

Reply:
if we are using string_view in the header itself, then yes, <string_view> should be included here for clarity (include what you use)